qgreenland/config/helpers/layers/lonlat.py

```diff
@@ -22,10 +21,10 @@ def _make_lonlat_layer(
 
     if asset.id.startswith('lat'):
         title_prefix = 'Latitude'
-        segment_max_distance = 1
+        # segment_max_distance = 1
     elif asset.id.startswith('lon'):
         title_prefix = 'Longitude'
-        segment_max_distance = 100
+        # segment_max_distance = 100
```

Comment:
I believe we still need to be segmentizing these lines; @trey-stafford ?

Reply:
That would be my guess. If it's working without I suppose we don't need it, but we should double-check the output to confirm.

If it's unnecessary lets remove this comment.